src/detect/ScanI2CTwoWire.cpp

```diff
@@ -256,7 +256,10 @@ void ScanI2CTwoWire::scanPort(I2CPort port)
                     type = BMP_280;
                 }
                 break;
-
+            case AHT10_ADDR:
+                LOG_INFO("AHT10 sensor found at address 0x%x\n", (uint8_t)addr.address);
+                type = AHT10;
+                break;
```

Comment:
See above comment, however this only applies if HAS_NCP5623 is defined.

Reply:
see above